wagtail/images/templates/wagtailimages/images/url_generator.html

```diff
@@ -35,8 +35,25 @@
         {% endpanel %}
 
         {% trans "URL" as heading %}
-        {% panel id="url" icon="link" heading=heading %}
-            <textarea class="w-image-url-generator__url" id="result-url" aria-labelledby="url-heading" rows="1" data-controller="w-action" data-action="focus->w-action#select"></textarea>
+        {% panel id="url" heading=heading %}
+            <div data-controller="w-clipboard w-clone" role="group" aria-labelledby="clipboardGroupLabel" data-w-clone-clear-delay-value="3500" data-action="w-clipboard:start->w-clone#clear w-clipboard:copied->w-clone#add w-clipboard:error->w-clone#add">
+                <label id="clipboardGroupLabel" hidden>Clipboard Actions</label>
```

Comment:
Thanks for labelling the input, a few notes here.

```suggestion
                <label class="w-sr-only" for="result-url">{% trans "Image URL" %}</label>
```

1. Please translate the label or use the existing `heading` variable.
2. You can use the label `for` attribute to associate the label to an input, remember to add `name` to the input though.
3. When using `hidden` this will hide for all things, even screen readers. Have a read of https://css-tricks.com/inclusively-hidden/ for more nuance. using the class `w-sr-only` should be what you need.
4. We usually use kebab-case for IDs / classes, so let's stick to that pattern for now.